### DISCUSSION TITLE
Access to articles that shaped the Earth Data Science Ideas

### DISCUSSION THREAD

**Discussion by oye-bobs** [cu-esiil-edu/2024-data-short-course#6] [General]
In the meeting yesterday, Nathan stated that he wanted to point out a couple of articles that really have shaped the approach to environmental data science education at Earth Lab and ESIIL, and that he'll put links to these articles in the discussion in Github so that we could have access to them, if we would like to. I would like to read these articles.

**Comment by nquarder** [comment]
Yes, great discussion post @oye-bobs and thank you for the reminder. Here are links to the articles that I mentioned in the talk yesterday: 

<a href="https://www.researchgate.net/profile/Rebecca-Hernandez-8/publication/318281425_Skills_and_Knowledge_for_Data-_Intensive_Environmental_Research/links/5c6f805c92851c6950368c71/Skills-and-Knowledge-for-Data-Intensive-Environmental-Research.pdf" target="_blank">Hampton et al, 2017 - Skills and Knowledge for Data Intensive Environmental Research</a>

<a href="https://esajournals.onlinelibrary.wiley.com/doi/full/10.1002/fee.2287" target="_blank">Farrell et al, 2021 - Training macrosystems scientists requires both interpersonal and technical skills</a>

<a href="https://academic.oup.com/bioscience/article/71/12/1274/6403634" target="_blank">Emery et al, 2021 - Data Science in Undergraduate Life Science Education: A Need for Instructor Skills Training</a>


Have a look at these materials and if you have questions or items that you would like to discuss further, please use this space here. -Nate Q

**Reply by oye-bobs** [reply]
Many thanks, @nquarder. After reviewing the articles, I've come to appreciate the significance of the skills you're imparting, which hold value not only in academia but also in various career paths.

**Reply by nquarder** [reply]
Agreed @oye-bobs - most of the students in our certificate program are professionals working in GIS or remote sensing.
